predicators/envs/pybullet_cover.py

```diff
@@ -193,6 +193,17 @@ def _reset_state(self, state: State) -> None:
                 assert self._held_obj_id == block_id
                 self._create_grasp_constraint()
 
+        # For any blocks not involved, put them out of view.
```

Comment:
this was just a bug before (but it only affected the unit tests, because in normal circumstances, there are always exactly 2 blocks in the cover env)